config/production_new.yml

```diff
@@ -0,0 +1,25 @@
+hint:
```

Comment:
this being here suggests master needs to be merged in? Is this https://github.com/mrc-ide/hint-deploy/pull/47/files?